tests/common/cts_selector.h

```diff
@@ -44,14 +44,14 @@ class cts_selector : public sycl::device_selector {
    *  return <  0  : device will never be selected
    *  return >= 0  : positive device rating
    */
-  virtual int operator()(const sycl::device& dev) const {
+  int operator()(const sycl::device& dev) const {
```

Comment:
In SYCL 2020 the device selector is just a simple callable so it could be just a simple function too.
So you could simplify the code everywhere by replacing the classes by just functions.
I do not know the impact on the use cases. If it does imply changing the use cases everywhere, just keep it with the classes.

Reply:
Turned cts_selector into function.
Only use case in kernel constructors test had to be updated. For other tests just code simplification.